tools/StaticAnalysis/Exceptions/AzureRM.RecoveryServices.SiteRecovery/BreakingChangeIssues.csv

```diff
@@ -16,3 +16,8 @@
 "Microsoft.Azure.Commands.RecoveryServices.SiteRecovery.dll","Microsoft.Azure.Commands.RecoveryServices.SiteRecovery.GetAzureRmRecoveryServicesAsrStorageClassification","Get-AzureRmRecoveryServicesAsrStorageClassification","0","1020","The cmdlet 'Get-AzureRmRecoveryServicesAsrStorageClassification' no longer has output type 'System.Collections.Generic.IEnumerable`1[Microsoft.Azure.Commands.RecoveryServices.SiteRecovery.ASRStorageClassification]'.","Make cmdlet 'Get-AzureRmRecoveryServicesAsrStorageClassification' return type 'System.Collections.Generic.IEnumerable`1[Microsoft.Azure.Commands.RecoveryServices.SiteRecovery.ASRStorageClassification]'."
 "Microsoft.Azure.Commands.RecoveryServices.SiteRecovery.dll","Microsoft.Azure.Commands.RecoveryServices.SiteRecovery.GetAzureRmRecoveryServicesAsrStorageClassificationMapping","Get-AzureRmRecoveryServicesAsrStorageClassificationMapping","0","1020","The cmdlet 'Get-AzureRmRecoveryServicesAsrStorageClassificationMapping' no longer has output type 'System.Collections.Generic.IEnumerable`1[Microsoft.Azure.Commands.RecoveryServices.SiteRecovery.ASRStorageClassificationMapping]'.","Make cmdlet 'Get-AzureRmRecoveryServicesAsrStorageClassificationMapping' return type 'System.Collections.Generic.IEnumerable`1[Microsoft.Azure.Commands.RecoveryServices.SiteRecovery.ASRStorageClassificationMapping]'."
 "Microsoft.Azure.Commands.RecoveryServices.SiteRecovery.dll","Microsoft.Azure.Commands.RecoveryServices.SiteRecovery.GetAzureRmRecoveryServicesAsrvCenter","Get-AzureRmRecoveryServicesAsrvCenter","0","1020","The cmdlet 'Get-AzureRmRecoveryServicesAsrvCenter' no longer has output type 'System.Collections.Generic.IEnumerable`1[Microsoft.Azure.Commands.RecoveryServices.SiteRecovery.ASRvCenter]'.","Make cmdlet 'Get-AzureRmRecoveryServicesAsrvCenter' return type 'System.Collections.Generic.IEnumerable`1[Microsoft.Azure.Commands.RecoveryServices.SiteRecovery.ASRvCenter]'."
+"Microsoft.Azure.Commands.RecoveryServices.SiteRecovery.dll","Microsoft.Azure.Commands.RecoveryServices.SiteRecovery.StartAzureRmRecoveryServicesAsrSwitchProcessServerJob","Start-AzureRmRecoveryServicesAsrSwitchProcessServerJob","0","3010","The property 'ChildErrors' of type 'Microsoft.Azure.Management.RecoveryServices.SiteRecovery.Models.HealthError' has been removed.","Add the property 'ChildErrors' back to type 'Microsoft.Azure.Management.RecoveryServices.SiteRecovery.Models.HealthError'."
```

Comment:
These all seem like actual breaking changes - you cannot ship breaking changes in this release.

Reply:
Properties added in breaking change is coming from API version bump up.
These properties are not used as input / output of powershell cmdlet